QgisModelBaker/libili2db/ilicache.py

```diff
@@ -213,7 +213,7 @@ def parse_ili_file(self, ilipath, encoding):
         Parses an ili file returning models and version data
         """
         models = list()
-        re_model = re.compile(r'\s*MODEL\s*([\w\d_-]+)\s.*')
+        re_model = re.compile(r'\s*MODEL\s*([\w\d_-]+)\s*[\(].*')
```

Comment:
```suggestion
        re_model = re.compile(r'\s*MODEL\s*([\w\d_-]+)\s*\(.*')
```
Brackets not needed.